app/views/users/show.html.erb

```diff
@@ -1,2 +1,9 @@
 <h1>User details page </h2>
 <p>Details of the user which includes both Bio and Posts </h2>
+<h2><%= @user.Name%></h2>
+<p><%= @user.Bio%></p>
+<p><%= @user.PostsCounter%></p>
+<%=link_to "Back", users_path%>
+<%=link_to "Posts", user_posts_path(@user)%>
```

Comment:
- [ ] Please make sure that your **See all Post** links takes to user's post index and is displaying like on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) as required for this project :+1: